docs/content/Schema/cube.md

```diff
@@ -300,18 +300,25 @@ cube(`OrderFacts`, {
 });
 ```
 
-`every` can accept only equal time intervals - so  "Day of month" and "month" intervals in CRON expressions are not supported.
+`every` can accept only equal time intervals - so "Day of month" and "month" intervals in CRON expressions are not supported.
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| Cube.js supports two different formats of CRON expressions: with leading seconds (6 parts) and leading minutes (5 parts).
```

Comment:
@ovr any chance we could add an example for both formats, please? 😀

Reply:
@hassankhan shall I add it to this warning or below?

Reply:
I think to the warning would be best; something like:

```
Cube.js supports two different formats of CRON expressions; with leading seconds (EXAMPLE_HERE) and with leading minutes (EXAMPLE_HERE).
```